src/Pos/Update/Poll/Pure.hs

```diff
@@ -0,0 +1,70 @@
+-- | Pure Poll
+
+module Pos.Update.Poll.Pure
+       ( PurePoll (..)
+       ) where
+
+import           Control.Lens               (to, (%=), (.=))
+import qualified Data.HashMap.Strict        as HM
+import qualified Data.HashSet               as HS
+import           System.Wlog                (CanLog, HasLoggerName, NamedPureLogger)
+import           Universum                  hiding (toList)
```

Comment:
It's more idiomatic to import `toList` from `Pos.Util.Modifier` qualified and not hide `toList` from `Universum`. All code is written this way.